### ISSUE TITLE
Cannot use identity.getRedirectURL()

### ISSUE THREAD

**Issue by kkoscielniak** [KeithHenry/chromeExtensionAsync#15]
Hello @KeithHenry, 

I'm trying to use `chrome-extension-async` in order to develop an extension which will use an OAuth login. 

When trying to: 
```javascript
const getRedirectUrl = async() => {
  const url = await chrome.identity.getRedirectURL();
  return url;
}; 
```

I get this uncaught error: 
```
Error: Invocation of form identity.getRedirectURL(function) doesn't match definition identity.getRedirectURL(optional string path)
```
Can you fix that?

----
Also, some workaround I've found for the future:
1) Remove `chrome-extension-async` temporarily
2) Invoke `chrome.identity.getRedirectURL()` in devtools console

**Comment by KeithHenry** [comment]
@kkoscielniak thanks for finding this - the problem is that `getRedirectURL` is included in the list of API functions to wrap, but it isn't a callback function.

I think the fix is to remove it from https://github.com/KeithHenry/chromeExtensionAsync/blob/128196eae7fdc99e34955bedaaa3310f14ebe1ea/chrome-extension-async.js#L181

PRs welcome! But otherwise I'll get it sorted tomorrow :-)

**Comment by KeithHenry** [comment]
This is fixed in 3.3.2, live on npm https://www.npmjs.com/package/chrome-extension-async

**Comment by kkoscielniak** [comment]
@KeithHenry Thanks!
